README.md

```diff
@@ -1,15 +1,41 @@
 Spacewalk playbooks and roles
 ==============
 
-For the lazy people: one command to install both ansible and run the playbook (requires you to be root):
+Changes:
+- Updated to Spacewalk version 2.9
+- Updated the iptables role
+- Admin account is automatically created
+- To enable a fully automated installation the password question has been removed.
+- Default admin password is admin
+- Default company name is Spacewalk
+- The default information can be changed in this file: ./roles/spacewalk-customisations/defaults/main.yml
+- Added support for Ubuntu Bionic
+
+Hardware / Virtual sizing advisement:
+- 1 CPU, 4GB of ram minimum, 64GB of disk 
+
+
+
+For the lazy people: 
+To install both ansible and the spacewalk server:
 ```
-sh <(curl -s https://raw.githubusercontent.com/fayaaz/spacewalk.ansible/master/init.sh)
+sh <(curl -s https://raw.githubusercontent.com/rhessing/spacewalk.ansible/master/init-server.sh)
```

Comment:
It would be nice if this pointed to this repo - but can be done in a separate PR

Reply:
yes this has to be done on a view places actually. Lets do that in a seperate PR